src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.VariableIdentifier.cs

```diff
@@ -11,6 +11,14 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class LocalDataFlowPass<TLocalState, TLocalFunctionState>
     {
+
+        protected static bool HasInitializer(Symbol field) => field switch
```

Comment:
Unnecessary blank line before the method, missing blank line after.

Reply:
Ah, yes, I used the "pull member to base" refactoring to do this and didn't look carefully at the results. I think this member maybe belongs in a different partial declaration of `LocalDataFlowPass`.

Reply:
Fixed